solutions/epi/lab3/kern/env.c

```diff
@@ -6,6 +6,7 @@
 #include <inc/string.h>
 #include <inc/assert.h>
 #include <inc/elf.h>
+#include <inc/stdio.h>
```

Comment:
What the purpose of this?

Reply:
Since assert.h includes stdio.h it is actually unnecessary. I probably included this before adding debug output